explainer.md

```diff
@@ -487,6 +489,17 @@ Appropriate places for a document badge could include:
 
 App badges are shown in OS-specific contexts. User agents should attempt reuse existing [operating system APIs and conventions](docs/implementation.md), to achieve a native look-and-feel for the badge.
 
+### Implementation Considerations for Platform Limitations
+
+Different operating systems have varying support for badge types. Since the OS ultimately controls badge rendering, it is the user agent that communicate the correct semantic intent:
+
+* **Flag support**: Some platforms do not natively support "flag" badges (badges without numbers). In such cases, user agents should communicate to the OS using the closest available representation that conveys badge presence (such as a generic symbol or the number "1") rather than requesting badge clearing.
```

Comment:
```suggestion
* **Flag support**: Some platforms do not natively support "flag" badges (badges without numbers). In such cases, user agents communicate to the OS using the closest available representation that conveys badge presence (such as a generic symbol or the number "1") rather than requesting badge clearing.
```